api/v2alpha1/pipeline_spec.proto

```diff
@@ -1104,6 +1104,24 @@ message PlatformDeploymentConfig {
   map<string, google.protobuf.Struct> executors = 1;
 }
 
+message WorkspaceConfig {
+  // Size of the workspace
+  // Example: "250Gi"
+  string size = 1;

```

Comment:
Can we also link to the k8s quantity docs for this: https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/quantity/